pyaedt/edb.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, edbpath=None, cellname=None, isreadonly=False, edbversion="20
                     self._messenger = EDBMessageManager(edbpath)
 
             self.student_version = student_version
-            self._messenger.add_info_message("Messenger Initialized in EDB")
+            self._messenger.add_info_message("_messenger Initialized in EDB")
```

Comment:
```suggestion
            self._messenger.add_info_message("Messenger Initialized in EDB")
```
In the info message I'd leave Messenger.